/usr/lib/python2.7/site-packages/celery/platforms.py:793: RuntimeWarning: You're running the worker with superuser privileges: this is
absolutely not recommended!

Please specify a different user using the -u option.

User information: uid=0 euid=0 gid=0 egid=0

  uid=uid, euid=euid, gid=gid, egid=egid,
[2018-04-28 16:55:14,031: INFO/MainProcess] Connected to amqp://guest:**@127.0.0.1:5672//
[2018-04-28 16:55:14,052: INFO/MainProcess] mingle: searching for neighbors
[2018-04-28 16:55:15,084: INFO/MainProcess] mingle: all alone
[2018-04-28 16:55:15,103: WARNING/MainProcess] /usr/lib/python2.7/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
[2018-04-28 16:55:15,103: INFO/MainProcess] celery@server155 ready.
[2018-04-28 16:55:32,439: INFO/MainProcess] Received task: hd_ansible.tasks.com_playbook[a1581eb2-40c0-4c09-97d1-1b2b18d8de98]  
 [WARNING]: Found variable using reserved name: hosts
 [WARNING]: Found variable using reserved name: roles
 [WARNING]: SELinux state change will take effect next reboot
[2018-04-28 16:56:17,080: INFO/PoolWorker-4] Task hd_ansible.tasks.com_playbook[a1581eb2-40c0-4c09-97d1-1b2b18d8de98] succeeded in 43.00676438s: {'status': 'SUCCESS', 'ansible_res': ['', 'PLAY [172.16.1.148] ************************************************************', '', 'TASK [init : 关闭selinux] ********************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 修改主机名] ************************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 更新系统] *************************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 关闭防火墙] ************************************************************', 'fatal: [172.16.1.148]: FAILED! => {"changed": false, "msg": "Could not find the requested service firewalld: host"}', '...ignoring', '', 'TASK [init : 关闭NetworkManager] *************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 安装ntpdate] ********************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 同步时间] *************************************************************', 'changed: [172.16.1.148]', '', 'TASK [i...', ...]}
[2018-04-28 17:26:47,355: INFO/MainProcess] Received task: hd_ansible.tasks.com_playbook[031404ff-c0be-4d1e-8f8d-2785f76e5905]  
 [WARNING]: Found variable using reserved name: hosts
 [WARNING]: Found variable using reserved name: roles
 [WARNING]: SELinux state change will take effect next reboot
[2018-04-28 17:27:30,068: INFO/PoolWorker-3] Task hd_ansible.tasks.com_playbook[031404ff-c0be-4d1e-8f8d-2785f76e5905] succeeded in 42.219671126s: {'status': 'SUCCESS', 'ansible_res': ['', 'PLAY [172.16.1.148] ************************************************************', '', 'TASK [init : 关闭selinux] ********************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 修改主机名] ************************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 更新系统] *************************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 关闭防火墙] ************************************************************', 'fatal: [172.16.1.148]: FAILED! => {"changed": false, "msg": "Could not find the requested service firewalld: host"}', '...ignoring', '', 'TASK [init : 关闭NetworkManager] *************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 安装ntpdate] ********************************************************', 'ok: [172.16.1.148]', '', 'TASK [init : 同步时间] *************************************************************', 'changed: [172.16.1.148]', '', 'TASK [i...', ...]}
[2018-05-08 10:52:10,410: INFO/MainProcess] Received task: hd_ansible.tasks.com_playbook[cd758ca8-ea04-438c-a25f-8271e9d3154f]  
ERROR! 'roles' is undefined
[2018-05-08 10:52:11,583: INFO/PoolWorker-4] Task hd_ansible.tasks.com_playbook[cd758ca8-ea04-438c-a25f-8271e9d3154f] succeeded in 0.916479692096s: {'status': 'Failed', 'ansible_res': []}
[2018-05-08 10:55:41,156: INFO/MainProcess] Received task: hd_ansible.tasks.com_playbook[64233008-18e9-46cb-890e-36493507f4cd]  
[2018-05-08 10:55:42,904: INFO/PoolWorker-3] Task hd_ansible.tasks.com_playbook[64233008-18e9-46cb-890e-36493507f4cd] succeeded in 0.0103061138652s: {'status': 'Failed', 'ansible_res': ''}
[2018-05-08 11:14:30,910: INFO/MainProcess] Received task: hd_ansible.tasks.com_playbook[26e103c2-c484-4f9c-a557-2f5e3fe96760]  
 [WARNING]: Found variable using reserved name: hosts
[2018-05-08 11:14:35,198: INFO/PoolWorker-4] Task hd_ansible.tasks.com_playbook[26e103c2-c484-4f9c-a557-2f5e3fe96760] succeeded in 3.07281958102s: {'status': 'SUCCESS', 'ansible_res': ['', 'PLAY [172.16.1.148] ************************************************************', '', 'TASK [Cammond execute] *********************************************************', 'changed: [172.16.1.148]', '', 'PLAY RECAP *********************************************************************', '172.16.1.148               : ok=1    changed=1    unreachable=0    failed=0   ', '']}
